src/main/kotlin/no/nav/k9/los/nyoppgavestyring/domeneadaptere/k9klagetillos/K9KlageTilLosAdapterTjeneste.kt

```diff
@@ -0,0 +1,340 @@
+            initialDelay = TimeUnit.SECONDS.toMillis(10),
+            period = TimeUnit.DAYS.toMillis(1)
+        ) {
+            if (kjørSetup) {
+                setup()
+            }
+            spillAvBehandlingProsessEventer()
+        }
+    }
+
+    private fun spillAvBehandlingProsessEventer() {
+        var behandlingTeller: Long = 0
+        var eventTeller: Long = 0
+        var forrigeOppgave: OppgaveV3? = null
+
+        log.info("Starter avspilling av BehandlingProsessEventer")
+        val tidKjøringStartet = System.currentTimeMillis()
+
+        val behandlingsIder = behandlingProsessEventKlageRepository.hentAlleDirtyEventIder()
+        log.info("Fant ${behandlingsIder.size} behandlinger")
+
+        behandlingsIder.forEach { uuid ->
+            transactionalManager.transaction { tx ->
+                val behandlingProsessEventer = behandlingProsessEventKlageRepository.hentMedLås(tx, uuid).eventer
+                behandlingProsessEventer.forEach { event -> //TODO: hva skjer om eventer kommer out of order her, fordi feks k9 har sendt i feil rekkefølge?
+                    val oppgaveDto = lagOppgaveDto(event, forrigeOppgave)
+
+                    val oppgave = oppgaveV3Tjeneste.sjekkDuplikatOgProsesser(oppgaveDto, tx)
+
+                    oppgave?.let {
+                        eventTeller++
+                        loggFremgangForHver100(eventTeller, "Prosessert $eventTeller eventer")
+                    }
+                    forrigeOppgave = oppgave
+                }
+                forrigeOppgave = null
+                behandlingTeller++
+                loggFremgangForHver100(behandlingTeller, "Forsert $behandlingTeller behandlinger")
+
+                behandlingProsessEventKlageRepository.fjernDirty(uuid, tx)
+            }
+        }
+        val (antallAlle, antallAktive) = oppgaveV3Tjeneste.tellAntall()
+        val tidHeleKjøringen = System.currentTimeMillis() - tidKjøringStartet
+        log.info("Antall oppgaver etter kjøring: $antallAlle, antall aktive: $antallAktive, antall nye eventer: $eventTeller fordelt på $behandlingTeller behandlinger.")
+        if (eventTeller > 0) {
+            log.info("Gjennomsnittstid pr behandling: ${tidHeleKjøringen / behandlingTeller}ms, Gjennsomsnittstid pr event: ${tidHeleKjøringen / eventTeller}ms")
+        }
+        log.info("Avspilling av BehandlingProsessEventer ferdig")
+    }
+
+    private fun loggFremgangForHver100(teller: Long, tekst: String) {
+        if (teller.mod(100) == 0) {
+            log.info(tekst)
+        }
+    }
+
+    private fun lagOppgaveDto(event: KlagebehandlingProsessHendelse, forrigeOppgave: OppgaveV3?) =
+        OppgaveDto(
+            id = event.eksternId.toString(),
+            versjon = event.eventTid.toString(),
+            område = "K9",
+            kildeområde = "K9",
+            type = "k9klage",
+            status = event.aksjonspunkttilstander.lastOrNull()?.status?.kode ?: "OPPR", // TODO statuser må gås opp
```

Comment:
forstått det som at det er én oppgave per aksjonspunkt...? Bør kanskje sortere disse på et tidspunkt eller id? For du kan ha flere OPPR aksjonspunkter samtidig (ihvertfall i k9-sak og k9-tilbake) - hva skjer hvis det finnes 3 ap med tilstand: 
```
[AVSL, OPPR, OPPR] 
``` 
også blir midterste avsluttet
```
[AVSL, AVSL, OPPR]
``` 
med denne koden skal midterste oppgaven avsluttes, men med koden her så vil den bli opprettet 🤔

Reply:
gravde litt videre i koden og ser at det kanskje er en oppgave per behandling, og at dette med oppgavestatus er under arbeid?

Reply:
men hvis det er én oppgave per behandling så vil status
```
[AVSL, OPPR, AVSL]
```
feilaktig sette status på oppgaven til AVSL.... 🤔

Reply:
trenger ikke å se på det nå, men vi har noe logikk på å utlede oppgave status her: 
https://github.com/navikt/k9-los-api/blob/bef6f062e1e030e796a0ed13416ff5da38edd762/src/main/kotlin/no/nav/k9/los/domene/modell/K9SakModell.kt#L43

tror det må på plass før vi kan få ut riktig statistikk? @stiangre

Reply:
/gruble
 
 Det er en oppgave pr aksjonspunkt, som du sier. Men den OppgaveDTOen vi lager her blir til en ny oppgaveVERSJON.
 
 Anyway, det blir bare OPPR hvis aksjonspunkttilstanden ikke har en status på seg. Ellers vil oppgaveversjonen få status lik aksjonspunktets status.